### PR TITLE
Fix: Prevent opening all folders on folder/file create

### PR DESCRIPTION
**Pull request by marek-mihok**

### CODE REVIEW

File: studio/studio.js [hunk 1]
```diff
@@ -125,10 +127,6 @@ const Tree = {
       this.folder.action = null
       window.parent.wave.emit('file_viewer', this.folder.isFolder ? 'new_folder' : 'new_file', { path: this.folder.path, name: this.folder.label })
     },
-    rename() {
-      this.folder.action = 'rename'
-      this.folder._labelCache = this.folder.label
-    },
```
Comment (mturoci) [inline studio/studio.js:131]: Why was this removed?

**Review by mturoci — CHANGES_REQUESTED** [review]
The fix looks good.

> Using v-show instead would preserve the state of all folders and it's sub-folders (like in VS Code).

Yes, definitely go for it. Also, let's open just the root folder on page load instead of opening them all.

File: studio/studio.js (hunk 1 above)
Comment (marek-mihok) [reply studio/studio.js:131]: It's just a leftover I forgot to remove while removing double click to rename feature implementation.

**Review by mturoci — APPROVED** [review]
